invenio.cfg

```
@@ -774,9 +774,14 @@ ZENODO_EOS_OFFLOAD_X509_KEY_PATH = ""
 """The path to the X509 private key file."""
 
 
-FILES_REST_DEFAULT_QUOTA_SIZE = 5 * 10**10
-FILES_REST_DEFAULT_MAX_FILE_SIZE = 5 * 10**10
+RDM_FILES_DEFAULT_QUOTA_SIZE = 5 * 10**10
+RDM_FILES_DEFAULT_MAX_FILE_SIZE = 5 * 10**10
```

```suggestion
RDM_FILES_DEFAULT_QUOTA_SIZE = 50 * 10**9     # 50GB
RDM_FILES_DEFAULT_MAX_FILE_SIZE = 50 * 10**9  # 50GB
```

nit: To make it easy to parse/read that it's 50GB